src/Autocomplete/src/Doctrine/EntityMetadata.php

```diff
@@ -41,22 +41,45 @@ public function isEmbeddedClassProperty(string $propertyName): bool
         return \array_key_exists($propertyNameParts[0], $this->metadata->embeddedClasses);
     }
 
-    public function getPropertyMetadata(string $propertyName): array
+    public function getFieldMetadata(string $propertyName): array
```

Comment:
Can you add phpdoc with `@return` for what this is an array of - e.g. `FieldMapping[]` if i'm correct.